_test/test_algorithms/test_sqw_eval.m

```diff
@@ -11,7 +11,7 @@
         sqw_2d_file_path = '../common_data/sqw_2d_1.sqw';
         sqw_2d_sqw_eval_ref_obj;
         sqw_2d_sqw_eval_ref_file = 'test_sqw_eval_gauss_ref.sqw';
-        sqw_2d_pix_pg_size = 3e5; % Gives us 6 pages
+        sqw_2d_pix_pg_size = floor(3e5/36); % Gives us 6 pages
```

Comment:
as we're making this change, be useful to document these magic numbers 3e5 and 36. We are only told that this ends up giving us 6 pages

Reply:
The underlying code is written by Tessella, and does not work properly. There is AFAIK ticket about it. As the code itself does not do what is requested, we need to decide, what to do about it. I am currently for full refactoring. 
Change comes from the fact I've changed definition of page size from bytes to pixels. As everything needs refactoring, a lot should change, so no point in concentrating on polishing the tests